profiles/package.mask/text.xml

```diff
@@ -14,6 +14,23 @@ have a comment detailing the specific reason for the mask. The format
 of the <c>package.mask</c> file is described in <c>man portage</c>.
 </p>
 
+<p>
+Development or unstable (per upstream declaration/categorization) versions of
+packages should usually be masked in <c>package.mask</c>. Upstreams may not
+deem such releases to be ready for general distribution (or safe to use), or
+may not be expecting bug reports from the wider userbase yet. The default
+should generally be to mask such versions, but it is acceptable to not mask
+in some circumstances <d/> e.g. upstream make very infrequent releases, the
+changes are safe (reviewed by the Gentoo maintainer), and perhaps other
```

Comment:
Do you really mean "and" or "or", here?

Reply:
I think I really did mean "and", but I think "or" is going to avoid ambiguity and it's clear you should exercise care when doing this. So I'll change to "or" for now.